weights/MuPogSF_UL/JSONtoROOT.py

```diff
@@ -0,0 +1,109 @@
+import json
+import numpy as np
+import ROOT as rt
+
+class TrgSF:
+    def __init__(self,name):
+        self.file_name=name
+        self.hist_name={}
+        self.data={}
+        self.nInputs={}
+        self.nScheme=0
+        self.__ReadJSON()
+        
+    def __ReadJSON(self):
+        f=open(self.file_name)
+        a=json.load(f)['corrections']
+        self.nScheme=len(a)
+        for iS in range(self.nScheme):
+            self.hist_name[iS]=a[iS]['name'].encode('utf-8')
+            self.nInputs[iS]=len(a[iS]['inputs'])-1
+            self.data[iS]=a[iS]
+            
+    def __GetHist2D(self,i):
+        data=self.data[i]
+        var1=data['inputs'][0]['name'].encode('utf-8')
+        var2=data['inputs'][1]['name'].encode('utf-8')
+        data=data['data']
+        bins1=np.array(data['edges'])
+        bins2=np.array(data['content'][0]['edges'])
+        nbin1=len(bins1)-1
+        nbin2=len(bins2)-1
+        hist_name=self.hist_name[i]+"_"+var1+"_"+var2
+        hist=rt.TH2F(hist_name,";|#eta|;p_{T} (GeV)",nbin1,bins1,nbin2,bins2)
+        hist_stat=rt.TH2F(hist_name+"_stat",";|#eta|;p_{T} (GeV)",nbin1,bins1,nbin2,bins2)
+        hist_syst=rt.TH2F(hist_name+"_syst",";|#eta|;p_{T} (GeV)",nbin1,bins1,nbin2,bins2)
+        for i1 in range(nbin1):
+            for i2 in range(nbin2):
+                nom=data['content'][i1]['content'][i2]['content'][5]['value']
+                stat=data['content'][i1]['content'][i2]['content'][3]['value']
+                syst=data['content'][i1]['content'][i2]['content'][4]['value']
```

Comment:
Why 5, 3, 4? The structure of the input files should be explained somewhere.

Reply:
added comments on important parts of JSON content